conf/modules.config

```diff
@@ -152,65 +121,88 @@ process {
     }
 
     withName: 'TABIX_PON' {
-        ext.when         = { !params.pon_tbi && params.pon && params.tools && (params.tools.contains('mutect2') || params.tools.contains('tnscope')) }
+        ext.when         = { !params.pon_tbi && params.pon && params.tools && params.tools.contains('mutect2') }
         publishDir       = [
             enabled: "${params.save_reference}",
             mode: params.publish_dir_mode,
             path: { "${params.outdir}/reference/pon" },
             pattern: "*vcf.gz.tbi"
         ]
     }
-}
+
+// PREPARE INTERVALS
+
+    withName: 'CREATE_INTERVALS_BED' {
+        publishDir       = [
+            enabled: "${params.save_reference}",
+            mode: params.publish_dir_mode,
+            path: { "${params.outdir}/reference/intervals" },
+            pattern: "*bed"
+        ]
+    }
+
+    withName: 'GATK4_INTERVALLISTTOBED' {
+        publishDir       = [
+            enabled: "${params.save_reference}",
+            mode: params.publish_dir_mode,
+            path: { "${params.outdir}/reference/intervals" },
+            pattern: "*bed"
+        ]
+    }
+
+    withName: 'TABIX_BGZIPTABIX' {
+        ext.prefix       = {"${meta.id}.bed"}
+        publishDir       = [
+            enabled: "${params.save_reference}",
+            mode: params.publish_dir_mode,
+            path: { "${params.outdir}/reference/target" },
+            pattern: "*bed.gz"
+        ]
+    }
 
 // BAM TO FASTQ
-process {
 
     withName: 'SAMTOOLS_FASTQ_MAPPED'{
-        ext.args2 = '-N'
+        ext.args2        = '-N'
         ext.prefix       = {"${meta.id}.mapped"}
     }
 
     withName: 'SAMTOOLS_FASTQ_UNMAPPED'{
-        ext.args2 = '-N'
+        ext.args2        = '-N'
         ext.prefix       = {"${meta.id}.unmapped"}
     }
 
     withName: 'SAMTOOLS_VIEW_MAP_MAP' {
-        ext.args = '-b -f1 -F12'
+        ext.args         = '-b -f1 -F12'
         ext.prefix       = {"${meta.id}.map_map"}
     }
 
     withName: 'SAMTOOLS_VIEW_MAP_UNMAP' {
-        ext.args = '-b -f8 -F260'
+        ext.args         = '-b -f8 -F260'
         ext.prefix       = {"${meta.id}.map_unmap"}
     }
 
     withName: 'SAMTOOLS_VIEW_UNMAP_MAP' {
-        ext.args = '-b -f4 -F264'
+        ext.args         = '-b -f4 -F264'
         ext.prefix       = {"${meta.id}.unmap_map"}
     }
 
     withName: 'SAMTOOLS_VIEW_UNMAP_UNMAP' {
-        ext.args = '-b -f12 -F256'
+        ext.args         = '-b -f12 -F256'
         ext.prefix       = {"${meta.id}.unmap_unmap"}
     }
-}
 
 // TRIMMING
-process {
     withName: 'TRIMGALORE' {
         ext.args         = '--fastqc'
-        ext.when         = { params.trim_fastq }
```

Comment:
Did you decide to now set this in the pipeline?

Reply:
that's it, for now it's better this way as we still have empty process if we do that

Reply:
ok then I will also remove the `when` statements from VC. Left them in for the future, but I would rather be consistent then half-way future proof things